core/modules/ccontrol/MergingHandler.h

```diff
@@ -45,6 +46,39 @@ namespace lsst {
 namespace qserv {
 namespace ccontrol {
 
+/// A class to delay creating the buffer until it is requested
+/// by xrootd SSI. The size of the buffer that will be needed is
+/// set using setTargetSize(int sz), and the buffer of that size is
+/// created by calling resizeToTargetSize(). When a buffer is no longer
+/// needed, zero() should be called to free the memory.
+class MergeBuffer {
```

Comment:
5-27 says a default constructor must be provided, and implicit copy constructors should be avoided.

This class should not be copied, so I deleted the copy copy constructor and operator=.